docs/guides/admin/docs/configuration/elasticsearch.md

```diff
@@ -0,0 +1,20 @@
+Elasticsearch Configuration
+===========================
+
+Elasticsearch powers the external API as well as the administrative user interface of Opencast.
+
+Installing Elasticsearch
+------------------------
+
+The [installation](../installation/index.md) documentation contains instructions for installing Elasticsearch along with your Opencast packaging of choice.
```

Comment:
Will you ever end up needing this if you end up here? You should have installed Opencast if you look at this and should have installed Elasticsearch in the process. You should never start here and then think that you need to install Opencast and Elasticsearch.